src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    static String horizontal = " __________________________________________________\n";
+    static String introduction = horizontal + "   Hello! I'm Duke\n" + "   What can I do for you?\n" + horizontal;
+    static String farewell = horizontal + "   Bye. Hope to see you again soon!\n" + horizontal;
+//    static Task[] taskList = new Task[100];
+    static ArrayList<Task> taskList = new ArrayList<>();
+    static int taskCount = 0;
+    static int descriptionIndex = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(introduction);
+
+        String line;
+        String[] splicedLine;
+        String commandWord;
+
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        String task;
+        String taskDate;
+
+        while (!line.equals("bye")) {
+
+            splicedLine = line.split(" ");
+            commandWord = splicedLine[0];
+
+            if ((commandWord.equals("done") || commandWord.equals("todo") || commandWord.equals("deadline") || commandWord.equals("event")) && splicedLine.length == 1) {
+                System.out.println(horizontal + "Oops! The description of " + commandWord + " cannot be empty!\n" + horizontal);
+            } else if (line.equals("list")) {
```

Comment:
The rest of the `else if` statements here check for matches to the commands - consider using switch case to make it more readable.